report/sections/Chapter 02 - Preliminaries/preliminaries.tex

```diff
@@ -91,6 +91,15 @@ \section*{PostgreSQL}
 Today Postgres features transactions with supports Atomicity, Consistency, Isolation, and Durability (ACID) properties, as well as automatic update of views, foreign keys, and procedures. 
 Furthermore modern Postgres supports a wide variety of deployment options from single machine usage to data warehouses and web services\cite{Postgres_Docs}.
 
-
-
-
+\section*{Usability testing}
+To evaluate a user interface, it can be useful to test its usability. ISO 9421 defines usability as
+\begin{quote}
+	"The effectiveness, efficiency and satisfaction with which specified users achieve specified goals in particular environments."
+\end{quote}
+
+Where effectiveness regards how well the users accomplished their goal, efficiency regards the resources expended to in relation to the completeness of the goals, and lastly, satisfaction regards the comfort of the system to the users.
+To test usability, one may perform a usability test. This is the study of users' interaction with a product. It is done to identify usability problems in a system, and results in a list of usability problems and knowledge of what works well.
```

Comment:
```suggestion
To test usability, one may perform a usability test. This is a study of the interaction that users have with a product. It is done to identify usability problems in a system, and results in a list of usability problems and knowledge of what works well.
```

Felt that this phrasing sounds a little better but that is up to interpretation.